core/src/main/java/bisq/core/support/dispute/agent/DisputeAgent.java

```diff
@@ -43,6 +43,12 @@
 public abstract class DisputeAgent implements ProtectedStoragePayload, ExpirablePayload {
     public static final long TTL = TimeUnit.DAYS.toMillis(10);
 
+    public enum DisputeAgentType {
+        ARBITRATOR,
+        MEDIATOR,
+        REFUNDAGENT
```

Comment:
We use underscore if it would be a camelCase'ed word: -> REFUND_AGENT

Reply:
The enum.name().lowercase value is meant to match the api registerdisputeagent (type) param 'refundagent', which is not camel case'd.

Reply:
Ah ok. I think to have a mapping method would be useful to void too thight coupling of cli-API to (new) Bisq domain code. Specially as code style is different for CLI and Java.

Reply:
But nothing urgent, can be done anytime later as well ....

Reply:
A solution was pushed in commit 35a77be.